ckanext/data_qld_theme/plugin.py

```diff
@@ -106,6 +106,22 @@ def populate_revision(resource):
         resource['revision_timestamp'] = current_revision.revision_timestamp
 
 
+def unreplied_comments_x_days(thread_url):
+    """A helper function for Data.Qld Engagement Reporting to highlight un-replied comments
+    after x number of days (number of days is a constant in the reporting plugin).
+    """
+    comment_ids = []
+
+    if 'data_qld_reporting' in config.get('ckan.plugins', False):
```

Comment:
Rather than hardcoding the plugin name, it would be better to just test for the presence of the toolkit action.

Reply:
Keeping in mind that this use case is specific to the Data.Qld implementation of reporting...

Either way you hard code a check for something. The fact that the plugin is enabled precludes that the action exists, otherwise another plugin could register the same action and it would be out of context for this use case.

This was included as a supplementary mechanism to assist the business requirements as a way of visually identifying those comments that have not had a reply since 10 days of their creation. If it has created an issue with progressing to merging, then we should remove it from the PR.